src/s3/util.ts

```diff
@@ -70,3 +70,17 @@ export function validateBucketName(name: string): string | undefined {
 
     return undefined
 }
+
+export function validateTime(time: string): string | undefined {
```

Comment:
Nit: this is only being used in `src/s3/commands/presignedURL.ts`, move the function there